src/components/bcros/filing/common/HeaderActions.vue

```diff
@@ -87,8 +87,6 @@
     <UModal v-model="isCommentOpen" :ui="{base: 'absolute left-10 top-5 bottom-5'}">
       <BcrosComment :comments="filing.comments" :filing="filing" @close="showCommentDialog(false)" />
     </UModal>
-
-    <BcrosLoadingModal :open="loadingDocuments" />
```

Comment:
This displayed as a code error because `loadingDocuments` isn't declared anywhere (and this component is missing the spinner-text prop).

According to Git Lens, this code was added to show the loading modal when downloading the Business Summary or expanding a ledger item and fetching its documents -- which both work fine without this code.